### ISSUE TITLE
project-switch-project: Marginalia not available on first run

### ISSUE THREAD

**Issue by manuel-uberti** [minad/marginalia#82]
Hi,

I am using the latest releases of `marginalia` and `vertico`. With this recipe, from `emacs -Q` on `C-x p p` I don't see Marginalia active:

- add a file `projects` to a Git versioned directory. In my case:
```
;;; -*- lisp-data -*-
(("~/.emacs.d/"))
```
- `emacs -Q`
- `M-x package-initialize`
- `M-x vertico-mode`
- `M-x marginalia-mode`
- `C-x p p`
 
![1](https://user-images.githubusercontent.com/8199224/119948665-56143b00-bf99-11eb-8335-4135ace7cc2e.png)

- Now press `RET` on `.emacs.d`
- Press `f`
- Press `RET` on the first candidate
- `C-x p p`
 
![2](https://user-images.githubusercontent.com/8199224/119948806-822fbc00-bf99-11eb-9ad5-fdc81d54834d.png)

**Comment by minad** [comment]
I assume `project-current` is nil in the first run. Can you confirm this?

**Comment by manuel-uberti** [comment]
You're correct, it's nil. On the second run it's `(vc . "~/.emacs.d/")`.

**Comment by minad** [comment]
Okay, so what should Marginalia do about this? There is no project, so no annotations :grinning:

**Comment by manuel-uberti** [comment]
Yes, I understand that. It's just "weird" UX not having them on the first try and then after working for a while on my projects having them. :)

We can close this then.

**Comment by minad** [comment]
Well, we could also decide for Marginalia to somehow "open" the project? How is this usually done?

**Comment by manuel-uberti** [comment]
I hit `RET` on the desired candidate of `C-x p p`.

**Comment by manuel-uberti** [comment]
But I guess is the following action, say pressing `f` for `project-find-files` that may be doing the actual "opening".
